### PR TITLE
Update timestamp setting

### PR DESCRIPTION
**Pull request by mrajashree**
Auth-service will update a timestamp setting in the setting table once it updates the encrypted auth config

### CODE REVIEW

**Comment by mrajashree** [comment]
https://github.com/rancher/rancher/issues/9457
https://github.com/rancher/rancher/issues/9455

File: server/auth_server.go [hunk 1]
```diff
@@ -503,11 +504,25 @@ func UpdateConfig(authConfig model.AuthConfig) error {
 	commonSettings[providerNameSetting] = authConfig.Provider
 	commonSettings[providerSetting] = authConfig.Provider
 	commonSettings[externalProviderSetting] = "true"
+	err = updateCommonSettings(commonSettings)
```
Comment (cjellick) [inline server/auth_server.go]: You used to do just a single call to updateCommonSettings. Why call it three separate times now?

Comment (mrajashree) [reply server/auth_server.go]: @cjellick the call to update the setting `api.security.enabled` should be after all other settings are updated without errors. That's how it was in the earlier code. But the call to update timestamp needn't be separate. I'll change this

**Comment by cjellick** [comment]
LGTM

**Pull request merged**